app/views/spree/admin/payments/index.html.haml

```diff
@@ -1,4 +1,4 @@
-= render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
+= render :partial => 'spree/admin/shared/order_page_title', locals: { current: 'Payments' }
```

Comment:
order tabs missing in payments page?

Reply:
Added back in, and syntax changed to remove rocket.